src/components/feed/index.js

```diff
@@ -6,32 +6,61 @@ import {LuShare} from "react-icons/lu"
 
 
 export function FeedItem(props){
-   
-    return(
-        <div>
-            <div className="flex">
+  
+  function formatDate(){
+    
+    const past = new Date('2023-07-07');
```

Comment:
why is the date hard-coded?
ideally, your component will have the date in there, which you can pass to your function.
tip: read about pure functions

Reply:
Resolved